NewHorizons/External/Modules/GameOverModule.cs

```diff
@@ -24,6 +24,33 @@ public class GameOverModule
         /// </summary>
         public string condition;
 
+        /// <summary>
+        /// The audio to use for the credits music. Can be a path to a .wav/.ogg/.mp3 file, or taken from the AudioClip list.
+        /// Note: only applies when creditsType is set to "custom".
+        /// </summary>
+        public string audio;
+
+        /// <summary>
+        /// The length of the fade in and out for the credits music.
+        /// Note: only applies when creditsType is set to "custom".
+        /// </summary>
+        [DefaultValue(1f)]
+        public float audioVolume;
```

Comment:
i think not going `= 1;` at the end makes this default to zero even tho the schema says default is 1